### PR TITLE
Adding nuxt import resolver for vue files. Change version to 1.3.0

### PR DESCRIPTION
**Pull request by marianobianchi**

### CODE REVIEW

File: packages/eslint-config-btrz/eslint-config-airbnb-base/rules/imports.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ module.exports = {
     'import/resolver': {
       node: {
         extensions: ['.mjs', '.js', '.json']
+      },
+      nuxt: {
```
Comment (lucianolev) [inline packages/eslint-config-btrz/eslint-config-airbnb-base/rules/imports.js]: @marianobianchi This setting should be in index.js I think. This file is a copy-paste from Airbnb's style guide and the idea is not to change it. Please check if you can move it and everything still works :)

**Review by lucianolev — CHANGES_REQUESTED** [review]

File: packages/eslint-config-btrz/eslint-config-airbnb-base/rules/imports.js (hunk 1 above)
Comment (marianobianchi) [reply packages/eslint-config-btrz/eslint-config-airbnb-base/rules/imports.js]: Great! I have changed and it is working ok

**Review by lucianolev — APPROVED** [review]
Perfect :)

**Review by lucianolev — CHANGES_REQUESTED** [review]
I realized that we should if is needed as a eslint-import-resolver-nux peer dependency.

**Review by lucianolev — APPROVED** [review]

**Pull request merged**